tool/net/help.txt

```diff
@@ -859,6 +859,10 @@ FUNCTIONS
    Slurp(filename:str) → str
            Reads file data from local file system.
 
+   Sleep(seconds:number)
+           Sleeps the specified number of seconds (can be fractional). The
+           smallest interval is a millisecond.
```

Comment:
Microsecond. If you want to nanosecond granular sleeps you could spinlock on `nowl()` using `asm("pause")` if `seconds < 2e-6`

Reply:
Funny typo; will fix in the next update (along with `1e6`).